Examples/Tests/ionization_dsmc/inputs_base_3d

```diff
@@ -21,7 +22,7 @@ warpx.const_dt = 1e-09
 geometry.dims = 3
 geometry.prob_hi = 0.1 0.1 0.1
 geometry.prob_lo = 0 0 0
-amr.max_grid_size = 8
+amr.max_grid_size = 4
```

Comment:
I noticed the test runs on 2 procs but with `max_grid_size=8` there was only 1 tile, so parallel execution was not really tested.